src/sql/cmd/cmd_utils.py

```diff
@@ -0,0 +1,12 @@
+import argparse
+import sys
+from sql import exceptions
+
+
+class CmdParser(argparse.ArgumentParser):
+    def exit(self, status=0, message=None):
+        if message:
+            self._print_message(message, sys.stderr)
+
+    def error(self, message):
```

Comment:
I don't think this is used, right?

Reply:
I left this part unchanged from the earlier version, it overrides the exit and error methods of `argparse.ArgumentParser` haven't test it but we are also handling the Usage errors through the validate function. Not sure if I should change it though!

Reply:
you're right. this was already there. I checked the source code and doesn't seem like it's being called anywhere so let's remove it

Reply:
made the change

Reply:
now we use `argparse.ArgumentParser()` as the parser